nntrainer/layers/multi_head_attention_layer.cpp

```diff
@@ -73,10 +73,317 @@ enum AttentionParams {
   transposed_attention_output,
 };
 
-void MultiHeadAttentionLayer::finalizeCommon(InitLayerContext &context) {}
+void MultiHeadAttentionLayer::finalizeCommon(InitLayerContext &context) {
```

Comment:
Do we need `finalizeCommon` else where?

Reply:
Nope. Removed finalizeCommon and merged it with finalize